airflow/www/static/foundation.min.css

```diff
@@ -0,0 +1,88 @@
+/*
```

Comment:
I checked both open source files. Highlight.js is under BSD 3 which should be ok based on the Apache link above.